core/src/parameter.jl

```diff
@@ -558,8 +558,13 @@ struct Parameters{T, C1, C2}
             edges_source::Dict{Int, Set{EdgeMetadata}},
             flow_dict::Dict{Tuple{NodeID, NodeID}, Int},
             flow::T,
+            flow_prev::Vector{Float64},
+            flow_integrated::Vector{Float64},
             flow_vertical_dict::Dict{NodeID, Int},
             flow_vertical::T,
+            flow_vertical_prev::Vector{Float64},
+            flow_vertical_integrated::Vector{Float64},
```

Comment:
I think it's fine to leave vertical flows out of this PR. I think we should remove them  altogether. For instance for a Basin, we'll need to integrate the separate vertical fluxes and put those in the Basin struct. Or do you rely on them in allocation?

Reply:
I think it's equally fine if allocation obtains the mean vertical fluxes from the basin struct.

Reply:
Ok yeah, I realize it wouldn't make sense to have both instantaneous and mean flows in flows.arrow, but I figured it would be ok if we follow up quickly. I'm fine either way!